target/linux/ath79/dts/tp9343_tplink_tl-wr941hp-v1.dts

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
```

Comment:
This should be moved before the other includes ...

Reply:
Done.